src/main/java/hudson/plugins/buildblocker/BlockingJobsMonitor.java

```diff
@@ -70,10 +69,10 @@ public BlockingJobsMonitor(String blockingJobs) {
         }
     }
 
-    public SubTask checkForBuildableQueueEntries(Queue.Item item) {
+    public Job checkForBuildableQueueEntries(Queue.Item item) {
```

Comment:
@felix you should use something like [this](https://github.com/jenkinsci/analysis-core-plugin/blob/9f8f492d2931981f91a06a8eb0123fd7d3cd1a58/src/main/java/hudson/plugins/analysis/views/ErrorDetail.java#L42) in all public methods that you changed the return value.